person.rb

```diff
@@ -21,4 +25,8 @@ def of_age?
   def can_use_services?
     @parent_permission || is_of_age?
   end
```

Comment:
- [ ] Please it would be better if you fix this unknown method `is_of_age?` which is not found in the Person class and makes the project throw an error when testing. I guess you want to can the `of_age` method in the Person class instead :pray:

Reply:
Thanks, must have slipped from me